GeometryTools/Polygon.swift

```diff
@@ -135,3 +135,22 @@ extension Polygon: Equatable {
         return lhs.vertices == rhs.vertices
     }
 }
+
+extension Polygon: CustomStringConvertible {
+
+    // MARK: CustomStringConvertible
+
+    /// Print the vertices in order of their appearance
+    public var description: String {
+        guard !vertices.isEmpty else { return "Polygon<0>[]" }
```

Comment:
I don't think we should be able to get here without having >= 3 vertices.

If we aren't `assert`-ing by this point, we should be.